app/views/users/show.erb

```diff
@@ -1,19 +1,31 @@
 
 
-<%= @user.username %>
+<h1><%= @user.username %>'s Profile Page </h1>
 <br>
-<% @user.questions.each do |question| %>
-<%= question.title %>
-<% end %>
+<b>Questions Asked:</b>
+<br>
+<% @questions.each do |question| %>
+<a href="/questions/<%=question.id%>"><%= question.title %> - Score:<%= question.score %></a>
```

Comment:
@DanielJPark 
Need an alternative filler here if the `@questions` variable is empty, like 

```
<div class="whoops"><p>No questions asked yet!</p></div>
```
Then we can style it differently if we put some kind of unique class on it. 

Otherwise, mine is rendering like this
![image](https://cloud.githubusercontent.com/assets/4001044/17595798/23979e32-5fb4-11e6-8855-6cc107d5c0cd.png)